### ISSUE TITLE
[Bug]: Trigger names in separate Babelfish databases conflict

### ISSUE THREAD

**Issue by swechsler** [babelfish-for-postgresql/babelfish_extensions#550]
### What happened?

If you have multiple Babelfish databases on a single instance, the following occurs:

```
Use DB1
go
create trigger foo on table1
after insert
as
select 1
go
use DB2
create trigger foo on table1
after insert
as
select 1
go
Msg 33557097, Level 16, State 1, Line 1
trigger "foo" already exists in the database
```
This is version 1.2

### Version

BABEL_1_X_DEV (Default)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

**Comment by robverschoor** [comment]
It is not supported to have multiple Babelfish instances on  single instance. There are various issues associated with that due to which this should not be done. if you somehow do this then all bets are off.
In fact, I thought creating multiple Babelfish databases was not possible. Could you please share the steps you took to create these?

**Comment by swechsler** [comment]
Simply run `create database` as many times as you need to. In fact, if multiple DBs are unsupported, why even allow you to create more than one?

**Comment by robverschoor** [comment]
I see what you mean, we have a terminology issue as well as a bug here.
As for terminology, 'babelfish database', this means: the PG database inside of which the SQL Server multi-DB structure is emulated. You need to call the sys.initialize_babelfish() function to initialize the Babelfish DB. There can be only one of these. In Aurora this PG DB is always called 'babelfish_db', in the OSS docs, the name 'demo' is used in the examples. You cannot actually see that you are in this PG database when connected to the TDS port (well, you can, by using a PG function that isn't actually a T-SQL function, but that aside). 
Once initialized, you can connect to the TDS port and create multiple T-SQL databases with CREATE DATABASE.

As for the bug: what we're hitting here is that PG has a uniqueness requirement around trigger names which is what we are hitting here. This is a known issue and it is being worked on. 
There is a similar issue with the uniqueness requirements of index names which has been resolved by renaming the indexes under the covers by appending a hash of the table/index name.

**Comment by robverschoor** [comment]
This will be fixed in the upcoming release 2.3.0

**Comment by suprio-amzn** [comment]
@swechsler: A fix to this issue is available in the current release ([Babelfish 2.3.0 for PostgreSQL 14.6](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/tag/BABEL_2_3_0__PG_14_6)) and has been available with this commit (https://github.com/babelfish-for-postgresql/babelfish_extensions/commit/29dbcbb8c53cbbd0d4bf987f9fc5d3f7448ad6de).
